src/server/actions.ts

```diff
@@ -28,6 +30,7 @@ const rpcTimeout = (service: string) => {
 export default class Actions {
   api: ApiPromise | undefined;
   account: KeyringPair | undefined;
+  #faucetBalance: number | undefined;
```

Comment:
What the purpose of using `#` here?

And can we declare it as optional with `?` instead of implicitly show undefined?

Reply:
But wait. How it works? How actions can have a balance? Why don't we use own class for balance or something like that?

Reply:
`#` makes the field private: [reference](https://www.typescriptlang.org/docs/handbook/classes.html#ecmascript-private-fields)

Reply:
> And can we declare it as optional with ? instead of implicitly show undefined?

We can but I kept a style similar to the existing code.